packages/video_player/video_player_avfoundation/ios/Classes/FLTVideoPlayerPlugin.m

```diff
@@ -3,8 +3,11 @@
 // found in the LICENSE file.
 
 #import "FLTVideoPlayerPlugin.h"
+#import "AVAssetTrackUtils.h"
```

Comment:
This should be down with `import messages.g.h`. Only the associated header goes in this first block, per the style guide.